mitiq/cdr/clifford_training_data.py

```diff
@@ -0,0 +1,516 @@
+import cirq
+from cirq.circuits import Circuit
+from numpy.random import choice, randint
+import numpy as np
+from typing import List, Tuple, Optional, Union
+
+
+# Global variable of Clifford angles in Rz gates:
+CLIFFORD_ANGLES = (0.0, np.pi / 2, np.pi, (3 / 2) * (np.pi))
+
+
+def generate_training_circuits(
+    circuit: Circuit,
+    num_training_circuits: int,
+    fraction_non_clifford: float,
+    method_select: str = "random",
+    method_replace: str = "closest",
+    random_state: Optional[Union[int, np.random.RandomState]] = None,
+    **additional_options: dict,
+) -> Tuple[List[Circuit], List[List[float]], List[List[float]]]:
+    """Function to return a list of near-Clifford circuits to create the
+    training data.
+
+    Args:
+        circuit: A circuit of interest.
+        num_training_circuits: Number of circuits in the returned training set,
+                               assumes already compiled into gate set
+                               (Rz, Rx, Z, X, CNOT)
```

Comment:
This is misplaced, right? The `circuit` is assumed to be compiled into this gate set, no?